File: plop-templates/adapter/acquiring/src/converter/request/EntryStateToStatusRequestConverter.java
package com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.converter.request;

import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.client.model.StatusRequest;
import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.model.CustomEntryStateModel;
import org.springframework.core.convert.converter.Converter;
import org.springframework.stereotype.Component;

@Component
public class EntryStateToStatusRequestConverter implements Converter<CustomEntryStateModel, StatusRequest> {

    @Override
    public StatusRequest convert(CustomEntryStateModel entryStateModel) {
        return new StatusRequest();
    }
}

File: plop-templates/adapter/acquiring/src/converter/request/EntryStateToStatusRequestConverter.kt
package com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.converter.request

import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.client.model.StatusRequest
import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.model.CustomEntryStateModel
import org.springframework.core.convert.converter.Converter
import org.springframework.stereotype.Component

@Component
class EntryStateToStatusRequestConverter : Converter<CustomEntryStateModel, StatusRequest> {

    override fun convert(entryStateModel: CustomEntryStateModel): StatusRequest {
        return StatusRequest()
    }
}
